storage/innobase/lock/lock0lock.cc

```diff
@@ -1186,46 +1177,12 @@ static void lock_rec_insert_to_waiting(hash_table_t *lock_hash, lock_t *lock,
   ut_ad(lock->is_waiting());
   ut_ad(rec_id.matches(lock));
   ut_ad(lock_mutex_own());
-<<<<<<< HEAD
-
-#ifdef WITH_WSREP
-  /* Disable CATS for galera for now */
```

Comment:
This should be also mentioned in the commit message.

Reply:
fixed